# contracts/test/ERC20ConfidentialUpgradeable_Harness.sol  ->  contracts/test/ERC20ConfidentialUpgradeable_Harness.fsol
# upstream 5138cb8 .. ported 4cefd22

diff --git a/contracts/test/ERC20ConfidentialUpgradeable_Harness.sol b/contracts/test/ERC20ConfidentialUpgradeable_Harness.fsol
index aa93535..e75eba8 100644
--- a/contracts/test/ERC20ConfidentialUpgradeable_Harness.sol
+++ b/contracts/test/ERC20ConfidentialUpgradeable_Harness.fsol
@@ -1,7 +1,7 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.25;
 
-import { ERC20ConfidentialUpgradeable } from "../ERC20Confidential/ERC20ConfidentialUpgradeable.sol";
+import { ERC20ConfidentialUpgradeable } from "../ERC20Confidential/ERC20ConfidentialUpgradeable.fsol";
 
 contract ERC20ConfidentialUpgradeable_Harness is ERC20ConfidentialUpgradeable {
     /// @custom:oz-upgrades-unsafe-allow constructor
